contracts/oracles/StkBNBOracle.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+import { OracleInterface } from "../interfaces/OracleInterface.sol";
+import { IPStakePool } from "../interfaces/IPStakePool.sol";
+import { ensureNonzeroAddress } from "@venusprotocol/solidity-utilities/contracts/validators.sol";
+import { EXP_SCALE } from "@venusprotocol/solidity-utilities/contracts/constants.sol";
+
+/**
+ * @title StkBNBOracle
+ * @author Venus
+ * @notice This oracle fetches the price of stkBNB asset
+ */
+contract StkBNBOracle is OracleInterface {
+    /// @notice Address of StakePool
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    IPStakePool public immutable STAKE_POOL;
+
+    /// @notice Address of stkBNB
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    address public immutable stkBNB;
+
+    /// @notice Address of Resilient Oracle
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    OracleInterface public immutable RESILIENT_ORACLE;
+
+    /// @notice Set this as asset address for native token on each chain.
+    address public constant NATIVE_TOKEN_ADDR = 0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB;
+
+    /// @notice Constructor for the implementation contract.
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address stakePool, address _stkBNB, address resilientOracleAddress) {
+        ensureNonzeroAddress(stakePool);
+        ensureNonzeroAddress(_stkBNB);
+        ensureNonzeroAddress(resilientOracleAddress);
+        STAKE_POOL = IPStakePool(stakePool);
+        stkBNB = _stkBNB;
+        RESILIENT_ORACLE = OracleInterface(resilientOracleAddress);
+    }
```

Comment:
If we are using underscore in the param name then we should use it in all the params.
```suggestion
    constructor(address _stakePool, address _stkBNB, address _resilientOracleAddress) {
        ensureNonzeroAddress(_stakePool);
        ensureNonzeroAddress(_stkBNB);
        ensureNonzeroAddress(_resilientOracleAddress);
        STAKE_POOL = IPStakePool(_stakePool);
        stkBNB = _stkBNB;
        RESILIENT_ORACLE = OracleInterface(_resilientOracleAddress);
    }
```